cmd/kubeadm/app/preflight/checks.go

```diff
@@ -435,15 +426,20 @@ func (sysver SystemVerificationCheck) Check() (warnings, errors []error) {
 	// Run the system verification check, but write to out buffered writer instead of stdout
 	bufw := bufio.NewWriterSize(os.Stdout, 1*1024*1024)
 	reporter := &system.StreamReporter{WriteStream: bufw}
+	runtimeOS := runtime.GOOS
```

Comment:
I'm not sure why you made this extra variable, things seem to be fine without it (using `runtime.GOOS` directly in this func)

Reply:
fixed